### PR TITLE
JW-991 cleanup

### PR DESCRIPTION
**Pull request by mclapinski**

### CODE REVIEW

File: rules.py [hunk 1]
```diff
@@ -1079,11 +1079,13 @@ def SetupBuildEnvironment(conf):
...
         env.Append(CCFLAGS = '/D_WINDOWS')
 
-        # Set Windows 10 as target system
+        # Set Windows Server 2016 as target system
         env.Append(CCFLAGS = '/D_WIN32_WINNT=0x0A00')
         # Set exception handling model
         env.Append(CCFLAGS = '/EHsc')
-        # Disable msvc paranoid warnings
+        # Disable min/max macros
+        env.Append(CCFLAGS = '/DNOMINMAX')
```
Comment (m-kostrzewa) [inline rules.py]: why is this needed? could you expand the comment here?

**Review by m-kostrzewa — APPROVED** [review]

**Review by katrybacka — APPROVED** [review]

**Comment by mclapinski** [comment]
Retest this please
